### DISCUSSION TITLE
Date picker support

### DISCUSSION THREAD

**Discussion by chrisj-3189** [flet-dev/flet#1143] [Q&A] [answered]
### Question

Does Flet have a built-in date picker control? 

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by iron3oxide** [comment, marked as answer]
Not yet, but there are issues and PRs relating to that. Some users have also built similar controls out of existing Flet controls, you can find those in the Discord server.

**Reply by chrisj-3189** [reply]
Thanks!

**Reply by iron3oxide** [reply]
No problem! If you have no further questions, please mark this discussion as answered.

**Reply by clstaudt** [reply]
@iron3oxide   What is the status of those pull requests?

**Reply by iron3oxide** [reply]
@clstaudt unfortunately they seem to be a little stale as their developer hasn't resolved the merge conflicts for some time now. See #981 and #948

**Reply by FeodorFitsner** [reply]
I've asked for status update in #948. I can take over that PR - it looks pretty finished.
